src/plugins/data/public/ui/filter_bar/filter_editor/index.tsx

```diff
@@ -345,7 +345,7 @@ class FilterEditorUI extends Component<Props, State> {
 
   private renderParamsEditor() {
     const indexPattern = this.state.selectedIndexPattern;
-    if (!indexPattern || !this.state.selectedOperator) {
+    if (!indexPattern || !this.state.selectedOperator || !this.state.selectedField) {
```

Comment:
logically it couldn't happen that we had `selectedOperator` without a `selectedField`, but I wanted to clarify this here because children now have `field` as required